### ISSUE TITLE
FlexPaper_1.3rc1  Exception: TypeError: Error #2007: Parameter blendMode must be non-null.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#83]
```
What steps will reproduce the problem?

When FlexPaper is running, following, exception is thrown
intermittently. Could not reproduce with consistency.

Couple of times, it was thrown when clicked on some link (not related
to flash). But when tried same sequence of actions again, it
wasn't reproducible. 

Will continue trying and provide more info as it becomes available. 
Please see exception below.

What is the expected output? What do you see instead?

Following Exception seen intermittently in Flash Player (ver 9)

TypeError: Error #2007: Parameter blendMode must be non-null.
    at flash.display::DisplayObject/set blendMode()
    at
mx.skins.halo::HaloFocusRect/updateDisplayList()[C:\autobuild\3.2.0\frameworks\p
rojects\framework\src\mx\skins\halo\HaloFocusRect.as:156]
    at
mx.skins::ProgrammaticSkin/validateDisplayList()[C:\autobuild\3.5.0\frameworks\p
rojects\framework\src\mx\skins\ProgrammaticSkin.as:421]
    at
mx.managers::LayoutManager/validateDisplayList()[C:\autobuild\3.5.0\frameworks\p
rojects\framework\src\mx\managers\LayoutManager.as:622]
    at
mx.managers::LayoutManager/doPhasedInstantiation()[C:\autobuild\3.5.0\frameworks
\projects\framework\src\mx\managers\LayoutManager.as:695]
    at Function/http://adobe.com/AS3/2006/builtin::apply()
    at
mx.core::UIComponent/callLaterDispatcher2()[C:\autobuild\3.5.0\frameworks\projec
ts\framework\src\mx\core\UIComponent.as:8744]
    at
mx.core::UIComponent/callLaterDispatcher()[C:\autobuild\3.5.0\frameworks\project
s\framework\src\mx\core\UIComponent.as:8684]



What version of the product are you using? On what operating system?

FlexPaper_1.3rc1 on WinXP (SP2) 
FlexBuilder: 3.x 
Flex SDK used for building SWF: 3.2.0

Please provide any additional information below.

Found following pointer :
URL: https://idnforums.intuit.com/messageview.aspx?catid=69&threadid=13454
Text: If you are getting this error, you need to update your Flex SDK to
3.4. This error is occurring because you are trying to compile your Flex
app with one version and some of the supplied SWC files are compiled with
3.4. This causes a version conflict. 


Thanks,
-darshan

```

Original issue reported on code.google.com by `darshanm...@gmail.com` on 28 May 2010 at 10:43

**Comment by GoogleCodeExporter** [comment]
```
Update:

Upgraded Flex SDK to 3.5.0.12683 and rebuilt. 

The exception seems to have stopped after this. Will update if happens again.

-darshan

```

Original comment by `darshanm...@gmail.com` on 28 May 2010 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the research & info Darshan! Will make sure next build is built with 
3.5
```

Original comment by `erik.eng...@gmail.com` on 30 May 2010 at 8:22
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.3 final
```

Original comment by `erik.eng...@gmail.com` on 6 Jun 2010 at 9:45
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hello. Firstly, a fantastic component. Congratulations.

Problem: I still get the blendMode cannot be null runtime error. I am using 
flex 3.5 and flexpaper 1.3.1. What are the parameter values for the blendMode 
property? I read elsewhere (google) that one of them is 'normal'. Can you 
explain what the blendMode property is all about?
```

Original comment by `ramvdi...@gmail.com` on 29 Jul 2010 at 1:43
